.codegen/model.go.tmpl

```diff
@@ -15,6 +16,16 @@ type {{.PascalName}} struct {
     {{- range .Fields}}
     {{.Comment "    // " 80}}
     {{.PascalName}} {{if .IsOptionalObject}}*{{end}}{{template "type" .Entity}} `{{template "field-tag" . }}`{{end}}
+
+	ForceSendFields []string `json:"-"`
```

Comment:
Since it's auto generated code and I want force send some fields, how do I define which fields to send?
It's not very clear from this PR at the moment

Reply:
Example for Clusters:
```
clusterDetails := compute.ClusterSpec{
		NumWorkers:   0,   
... //other details
                ForceSendFields: []string{"NumWorkers"} 
}
```
We can add a comment to it. How does this sound?
```
// Fields to be send as part of the request even if they are empty or have the default value. Null elements will still not be send.
```

Reply:
SDK users are interacting with the SDK names. For them the JSON names are almost an implementation details (although they see them in errors.)